app/assets/stylesheets/application.css

```diff
@@ -5,6 +5,7 @@
   --light-grey: rgb(227, 227, 227);
   --pure-white: white;
   --pure-black: black;
+  --button-green: #43B73A;
 }
```

Comment:
| Microverse Button | Your Button |
|-|-|
| ![image](https://github.com/firepicaso/ror-budget-app/assets/93039359/c2710a37-88fe-4e8b-aeeb-b83bf8f9ee18) | ![image](https://github.com/firepicaso/ror-budget-app/assets/93039359/da51e4ae-80a5-429b-9fea-38c10dc22860) |
| Original Color |
| ![image](https://github.com/firepicaso/ror-budget-app/assets/93039359/5be67e3c-a66c-4223-812d-b170da077c86) |

- [ ] Great work on making the button well but it would be if you use the right color to follow the wireframe :+1: as suggested by the previous reviewer :+1: .
- [ ] It would be nice If you added the white inside border of the button which you can see in the wireframe to make your design look professional :blush: .